MaeConstants.hpp

```diff
@@ -21,5 +21,38 @@ const char* const BOND_ATOM_1 = "i_m_from";
 const char* const BOND_ATOM_2 = "i_m_to";
 const char* const BOND_ORDER = "i_m_order";
 
+/**
+ * These are the prefixes used to store stereochemical properties in Maestro
+ * files. Each of those properties will be associated to one of these prefixes
+ * plus an integer (integers don't have any particular meaning, they just
+ * prevent the property names from clashing), e.g.:
+ *
+ * s_st_Chirality_1 :        2_R_1_3_5_4
+ * s_st_AtomNumChirality_4:  6_ANR_1_5_7
```

Comment:
We're not going to use this property name anymore (see https://jira.schrodinger.com/browse/SHARED-6609). So this should be:

    s_st_Chirality_1 :        2_R_1_3_5_4
    s_st_Chirality_2 :        6_ANR_1_5_7

Reply:
Yeah, but what about backwards compatibility? We should be able to correctly read existing files, as well as those generated by older versions of the Schrodinger Suite.